.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Run the build
+        run: npm run build
+
+      - run: npm version patch
+
+      - run: npm login --scope=<your-scope> --registry=https://registry.npmjs.org/
```

Comment:
I guess the `--scope` should be set or omitted.

Reply:
This is just draft version and I should delete this.